src/Calculator/Controls/CalculationResult.cpp

```diff
@@ -228,7 +237,7 @@ void CalculationResult::UpdateTextState()
             m_textBlock->InvalidateArrange();
             return;
         }
-        assert(m_textBlock->FontSize >= MinFontSize && m_textBlock->FontSize <= m_startingFontSize);
+        assert(m_textBlock->FontSize >= MinFontSize && m_textBlock->FontSize <= MaxFontSize);
```

Comment:
This assert is failing for me under these circumstances:
1. Open currency converter
2. Type in all 5s
3. Grab the right side of the window and drag it left to make the window even smaller
4. Dev version of calc crashes since the assert is false

I noticed a few times it takes a few resizes to get it to fail.

![image](https://user-images.githubusercontent.com/6334170/54165716-24961000-441f-11e9-82cf-4928a62d87b0.png)

Reply:
The issue existed before, but the next commit will fix it! Nice found